modules/python/pylib/syslogng/debuggercli/__init__.py

```diff
@@ -21,11 +21,31 @@
 #############################################################################
 
 from __future__ import absolute_import
-from syslogng.debuggercli.readline import setup_readline
 
+def is_readline_available():
+    try:
+        import readline
+    except ImportError:
+        return False
+    return True
+
+def is_editline_available():
+    try:
+        import editline
+    except ImportError:
+        return False
+    return True
+
+def setup_read_or_editline():
```

Comment:
Just a small note: I thought it would be nice to move `__setup_performed__` to this level, to avoid code duplication. So I started to tweak this part.
If none of the setup succeeded, we only lost tab completion, and everything else works fine! Which is really good.